content/creator/scenes/releases/version-support-agreement.md

```diff
@@ -0,0 +1,71 @@
+
+* *Version family* -- all versions that have identical major version form a family. We call version A a *successor* of version B if A and B belong to same family and A minor number is higher than B's (for example, 7.11.0 is a successor of 7.10.0).
+
+* *Breaking change* -- a change that forces a user to change their code or assets to keep them in functioning state. For example, a property changes name, and forces the user to change that property name every time it's used throughout their code.
+
+* *Non-breaking change* -- a change that does not require any action by a user, the behavior and properties of the user's code and assets are unchanged if they migrate their scene.
+
+* *Stable version* -- a version that forbids breaking changes in all its successors (which are all considered as stable too). Any breaking change must be introduced only through creation of a new version family by incrementing the major version number. Non-breaking changes are reflected through incrementing the minor version number.
+
+* *Unstable version* -- a version that allows breaking changes in its successors. Breaking changes may be introduced by incrementing the minor version number.
+
+## Support policy
+
+In every stable version family, the Decentraland Fountation supports only the latest minor version. At any given time there should be at most two supported families. 
+
+For example, if the latest minor version of the 6.x family is 6.11.11, and the latest minor version of the 7.x family is 7.3.1, content creators are expected to develop their scenes on either 6.11.11 or 7.3.1. Scenes that were developed and published with version 6.11.10 or older should keep working and players will be able to enjoy them. However, if these older scenes experience any issue, they must first be updated to a supported version, and the issue will only be investigated if it still occurs in a supported version.
+
+The Decentraland Editor automatically updates the renderer package of all scenes within a same version familiy, so that all developers are using the latest supported version when they develop their scenes.
+
+## Feature development
+
+New features will only be released into the latest version in development. As soon as the development team starts work in a new version family, older version families that remain in support will only receive major bugs fixes, and no additional features will be implemented.
+
+New features are released on **Minor version** updates, **Patch version** updates should only include bug fixes.
+
+## Breaking changes
+
+Breaking changes should only occur in major releases. There will be no breaking changes within the stable minor releases of a same version family. New releases will extend the capabilities of the existing syntax, but never change what the established syntax produces, except when fixing bugs.
```

Comment:
"There will be no breaking changes" and "never change what the established syntax produces" is a bit too strong wording. Unanticipated crippling emergencies that could only be fixed via breaking change may arise.

We need a loophole to address hypothetical emergency situations without introducing a new major version, if a breaking change is relatively minor. Saying something like "breaking change may be introduced in a minor version in case of emergency and absence of any other means to address it, however we'll do our best to avoid such drastic measures".

See here for an example from Google: https://google.aip.dev/181

Reply:
reworded in the last commit